src/task/CamelKTaskDefinition.ts

```diff
@@ -20,6 +20,8 @@ import * as kamel from '../kamel';
 import * as vscode from 'vscode';
 import * as IntegrationUtils from '../IntegrationUtils';
 
+export const TASK_NAME_DEFAULT_DEPLOY_FROM_ACTIVE_EDITOR = "Start in dev mode Camel K integration opened in active editor";
```

Comment:
I find that label a bit strange. Wouldn't it be better to name it like:
`Start currently opened Camel K integration in dev mode` ?

Reply:
maybe. This has not been modified in this PR. Creating an issue to discuss to provide a better naming https://issues.redhat.com/browse/FUSETOOLS2-785